server/tests/follower.test.js

```diff
@@ -0,0 +1,149 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import faker from 'faker';
+import app from '../app';
+
+chai.use(chaiHttp);
+const { expect } = chai;
+
+let firstUserToken, secondUserToken, firstUserId, secondUserId;
+const fakerObject = () => {
+  const username = faker.random.alphaNumeric(6);
+  const email = faker.internet.email();
+  const password = faker.internet.password();
+  return { username, email, password };
+};
+
+describe('TESTS TO CREATE USERS', () => {
```

Comment:
why exactly are you testing creation of new users again?

Reply:
I wanted to grab the token.